### ISSUE TITLE
UX: User feedback for wrong selection

### ISSUE THREAD

**Issue by rugk** [codedust/wahlsalon19-werhatsgesagt#1]
My UX experience just now:
1. clicked on party logo
2. it has been selected

So I thought: Okay, maybe I can select multiple ones, if it has been said by multiple parties.

Then:
3. But how do I now confirm my selection? get the solution?

-> Click on "next quote" possibly…

I only noticed when I accidentally chose the right party, that I was actually sopposed to continue guessing until I have it.

## Solution

* make it less "selection" if you click on it, but really show "wrong answer"/"wrong guess" or a cross or so
* or just solve it after one click, I don't think I really need to continue guessing after my 1st wrong guess

**Comment by codedust** [comment]
Thanks for your feedback! Do you have any ideas how to realize your first suggestion? I'd favor it over the second idea since having only one chance to guess the correct party might be quite frustrating.

**Comment by rugk** [comment]
Yeah, the simplest thing would just to change the design of the "selected" party image: Add an overlay with a cross or so. (CSS background or so may be working)
Or show a text message somewhere.
Or make the party image go away… or… or… or…
